test/testCSVParser.cc

```diff
@@ -128,7 +128,45 @@ SUITE(CSVParser)
       reportFromCSVFile(isn,osn,*this);
       
       CHECK(osn.str().find("missing numerical data") != std::string::npos);
-    }    
+    }
+    
+   TEST_FIXTURE(CSVDialog,guessSpaceFile)
+    {
+      spec=DataSpec();
+      string url="../test/testEqGodley.csv";
```

Comment:
This will break the test/00/unittest.sh regression test. The unittest executable is executed out of some random directory, not in the repo directory. Any supporting files, such as this one, is copied into the current directory before running, so you can just refer to the unqualified name.

Reply:
I've changed it to just "testGodley.csv" and it works as you've pointed out, thanks.